src/Storefront/composer.json

```diff
@@ -34,6 +34,7 @@
         "php": "^7.4.3 || ^8.0",
         "cocur/slugify": "4.0.0",
         "doctrine/dbal": "2.13.3",
+        "guzzlehttp/psr7": "^1.7 || ^2.0",
```

Comment:
The dependency has also to be added to the root `composer.json` file.

Additionally i've seen that the core package also has a dependency on the psr7 package, but only requires the `guzzle-http/guzzle` package.

can you add the required package there too?

Reply:
@keulinho I can but I think it is only used directly in Storefront, if so adding it to storefront only should be enough - but I will re-check (though probably not before Monday)

Reply:
@keulinho I did check now after all and you are correct, core and storefront currently both use guzzelhttp/psr7 directly, so I added it in both and in the root composer.json